### PR TITLE
C#: Dapper support

### PR DESCRIPTION
**Pull request by tamasvajk**
[Differences job](https://jenkins.internal.semmle.com/job/Changes/job/CSharp-Differences/985/)

### CODE REVIEW

File: csharp/ql/src/semmle/code/csharp/frameworks/Dapper.qll [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Classes for modelling Dapper.
```
Comment (intrigus-lgtm) [inline csharp/ql/src/semmle/code/csharp/frameworks/Dapper.qll:2]:
```suggestion
 * Classes for modeling Dapper.
```

File: csharp/ql/src/semmle/code/csharp/frameworks/Sql.qll [hunk 2]
```diff
@@ -5,6 +5,8 @@ private import semmle.code.csharp.frameworks.system.Data
 private import semmle.code.csharp.frameworks.system.data.SqlClient
 private import semmle.code.csharp.frameworks.EntityFramework
 private import semmle.code.csharp.frameworks.NHibernate
+private import semmle.code.csharp.frameworks.Dapper
+private import semmle.code.csharp.dataflow.DataFlow3
```
Comment (hvitved) [inline csharp/ql/src/semmle/code/csharp/frameworks/Sql.qll:9]: Better to use `DataFlow4` to avoid overlap with `Xml.qll`.

**Review by hvitved — COMMENTED** [review]
LGTM. We should probably add a change note.

File: csharp/ql/src/semmle/code/csharp/frameworks/Dapper.qll (hunk 1 above)
Comment (tamasvajk) [reply csharp/ql/src/semmle/code/csharp/frameworks/Dapper.qll:2]: Thanks. We seem to be using both variants, but it would make sense to unify these.

**Review by hvitved — APPROVED** [review]

**Pull request merged**